src/lapack/f2c.h

```diff
@@ -22,11 +22,13 @@
 #include "_hypre_utilities.h"
 #include "math.h"
 
+/*
```

Comment:
Why did you comment this out?  Why not use it instead and redefine with 'hypre_sqrt'?

Reply:
Good point.